docs/application/native/guides/app-management/tidl.md

```diff
@@ -19,6 +19,9 @@ Usage:
   -o, --output=OUTPUT         The generated interface file.
   -p, --proxy                 Generate proxy code.
   -s, --stub                  Generate stub code.
+  -r, --rpclib                Generate C# library for rpc-port (Since Tizen 6.0).
+  -t, --thread                Generate thread code with stub code (Since Tizen 6.0).
+  -b, --beta                  Use beta version for private file sharing (Since Tizen 6.0).
```

Comment:
Question. This section seems to use the actual help message from TIDLC. 
If so, are `(Since Tizen 6.0)` messages also included in the help message?

If not, how about removing `(Since Tizen 6.0)` messages from the code block, and add a note that describing -r, -t, -b options are supported since Tizen 6.0, after the code block?

Reply:
I've added the descriptions after the code block.
And, "Generate C# library for rpc-port (Since Tizen 6.0)" is my mistake. It's supported since Tizen 4.0.